ocean_grid_generator.py

```diff
@@ -1077,7 +1077,7 @@ def main(
     ###
     ###Southern Ocean grid
     ###
-    lat0_SO = -78.0  # Starting lower lat of Southern Ocean grid
+    lat0_SO = -81.0  # Starting lower lat of Southern Ocean grid
     if south_ocean_lower_lat > -90:
         lat0_SO = south_ocean_lower_lat
```

Comment:
Can we not use the `southern_ocean_lower_lat` argument instead of changing this ?

Reply:
Updated in this commit d9799fd